envs/create_conda_environments.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
This looks great, I'm not sure if it's best practice to recommend people to run a shell script for installation though from a security standpoint. Maybe add something to the documentation to check scripts first before you install them?

Reply:
I wasn't aware that was an actual issue. Would removing the shebang line solve this concern? I instruct users to call the script with bash anyway, so that shouldn't be a problem.

Reply:
I meant the comment for the whole file but actually it's fine since it's not run with root privileges.

Reply:
I was thinking the same. Let's keep it as is then 👍